sci-physics/pythia/pythia-8.3.09.ebuild

```diff
@@ -13,13 +13,13 @@ DESCRIPTION="Lund Monte Carlo high-energy physics event generator"
 HOMEPAGE="https://pythia.org/"
 SRC_URI="https://pythia.org/download/${PN}${MV//./}/${MY_P}.tgz
 	test? ( lhapdf? (
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/CT10.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/MRST2007lomod.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/NNPDF23_nlo_as_0119_qed_mc.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/NNPDF23_nnlo_as_0119_qed_mc.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/cteq66.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/cteq6l1.tar.gz
-		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/unvalidated/MRST2004qed.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/CT10.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/MRST2007lomod.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/NNPDF23_nlo_as_0119_qed_mc.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/NNPDF23_nnlo_as_0119_qed_mc.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/cteq66.tar.gz
+		https://lhapdfsets.web.cern.ch/lhapdfsets/current/cteq6l1.tar.gz
+		https://www.hepforge.org/downloads/lhapdf/pdfsets/v6.backup/${LHA_VER}/MRST2004qed.tar.gz
```

Comment:
Can the tests maybe run with the split MRST2004qed_proton MRST2004qed_neutron files? The split files are available in https://lhapdfsets.web.cern.ch/lhapdfsets/current/.

Reply:
Yeah, I'll check where it is needed and if it works with the other ones.